src/schema/resolvers/signin.js

```diff
@@ -0,0 +1,31 @@
+
+const INVALID_CREDENTIALS_ERROR_MESSAGE = 'Invalid username and/or password'
+
+export default {
+  Mutation: {
+    async signin (parent, { data: { username, password: passwordFromClient } }, context) {
+      const user = await context.models.user.findOne({ where: { username } })
+
+      if (user) {
```

Comment:
These `if/else` may be avoided with a single `try/catch` block. The code may even be clearer that way.

```
try {
  ....
} catch (err) {
  throw new AuthenticationError(err.message)
}
```

Then, in Express you could add a generic error handler that translates error _types_ into error _messages_ suitable for users.